handlers/aws/handler.py

```diff
@@ -85,6 +86,13 @@ def lambda_handler(lambda_event: dict[str, Any], lambda_context: context_.Contex
         shipper_cache: dict[str, CompositeShipper] = {}
         for replay_record in lambda_event["Records"]:
             event = json_parser(replay_record["body"])
+
+            decoded, ok = try_base64_decode(event["event_payload"])
+            if ok:
```

Comment:
What if it is not ok? Is it possible?

Reply:
I see that compressing can still fail with:

```
def try_base64_decode(data: str) -> tuple[bytes, bool]:
    try:
        decoded = base64.b64decode(data, validate=True)
        return decoded, True
    except Exception:
        return bytes(), False
```

So if compressing and encoding failed for some reason, we hit the non ok. Do you think it is worth continuing it in this case?

Reply:
I reviewed the ESF architecture, and it seems possible that we can put messages on the replay queue manually. I don't know if the user would ever do it (well, if it is possible, it could happen, I guess), not sure if it is worth considering.

Reply:
> What if it is not ok? Is it possible?

One possible reason is that users upgrade to this new ESF version, but the SQS queue still contains messages sent with the previous ESF version in plain text.